packages/engine/tests/units/state/edit/dot/mod.rs

```diff
@@ -1,9 +1,23 @@
-use crate::run_test;
+mod js {
+    use crate::run_test;
 
-run_test!(bool);
-run_test!(bool_array);
-run_test!(number);
-run_test!(number_array);
-run_test!(string);
-run_test!(object);
-run_test!(object_array);
+    run_test!(bool, JavaScript);
```

Comment:
Only just noticed this, is it a problem that we're using `bool` here? 

I mean I guess not because they've already passed but it does seem a little strange to use a type as something that isn't a type (if that makes sense)

Reply:
This shouldn't be a problem, but we also use `object` instead of `struct`. May we should just rename it to `boolean`?

Reply:
Hm, my inclination is that we should change it because it just feels wrong to shadow `bool`. But also we'd have to rename all of the other` bool_` tests for consistency and it'll make it a lot wordier. Lets just leave it if it's working and not causing problems